sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/reflect/ByteBuddyDoFnInvokerFactory.java

```diff
@@ -432,7 +432,7 @@ public static <InputT, OutputT> double invokeGetSize(
             .getProgress()
             .getWorkRemaining();
       } else {
-        return 1.0;
+        return 0.0;
```

Comment:
Maybe adding a comment with the rationale for returning 0 will be useful for the next person

Reply:
Please update the comment where the interface is defined, as it explicitly states 1.0.  The comment requested above would be better there too.

Reply:
I'm referring to the DoFnInvoker.invokeGetSize comment